server/src/main/java/dev/slimevr/vr/trackers/udp/UDPPacket21Reset.java

```diff
@@ -0,0 +1,32 @@
+package dev.slimevr.vr.trackers.udp;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+
+public class UDPPacket21Reset extends UDPPacket {
```

Comment:
Can you please rename the packet to UDPPacket21UserAction and update action IDs so that RESET starts from 2 insted of 0. This will allow the system to be expanded in the future, I have a few ideas for it and will create a ticket later.